build.sbt

```diff
@@ -0,0 +1,3 @@
+Project.defaultSettings
```

Comment:
"That's how I found it" :) Good point, `defaultSettings` includes the other. I'll remove `Project.defaultSettings` in a sec.